### DISCUSSION TITLE
Power Fx in Power Automate Desktop

### DISCUSSION THREAD

**Discussion by MikeStall** [microsoft/Power-Fx#2120] [General]
Power Fx is now in preview for Power Automate Desktop! 

https://powerautomate.microsoft.com/en-us/blog/december-2023-update-of-power-automate-for-desktop/

**Comment by jas-valgotar** [comment]
[celebrate]     Jas Valgotar reacted to your message:
________________________________
From: Mike Stall ***@***.***>
Sent: Thursday, December 21, 2023 7:30:55 AM
To: microsoft/Power-Fx ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/Power-Fx] Power Fx in Power Automate Desktop (Discussion #2120)


Power Fx is now in preview for Power Automate Desktop!

https://powerautomate.microsoft.com/en-us/blog/december-2023-update-of-power-automate-for-desktop/
